infrastructure/task_definition.tf

```diff
@@ -16,12 +16,43 @@ resource "aws_ecs_task_definition" "veda_task_definition" {
           }
       }
     }
+
   ])
   family                   = "${var.prefix}-tasks"
   requires_compatibilities = ["FARGATE"]
   network_mode             = "awsvpc"
+  execution_role_arn = module.mwaa.mwaa_role_arn
+  task_role_arn = module.mwaa.mwaa_role_arn
   cpu                      = 1024
   memory                   = 2048
+}
+
+
+resource "aws_ecs_task_definition" "veda_vector_task_definition" {
+
+
+  container_definitions = jsonencode([
+
+    {
+      name      = "${var.prefix}-veda-vector_ingest"
+      image     = "${local.account_id}.dkr.ecr.${local.aws_region}.amazonaws.com/${var.prefix}-veda-vector_ingest"
+      essential = true,
+      logConfiguration = {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": module.mwaa.log_group_name,
```

Comment:
non-blocking question for @amarouane-ABDELHAK : noticed that `module.mwaa.source` is pointing to a private repo. Can we move that into "NASA-IMPACT" by chance?

Reply:
Yes, I am planning to open a PR to point it to NASA-IMPACT after this PR